### ISSUE TITLE
NVDA 2026.1.1 not on NV Access download page

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#20244]
### Detailed description of the issue
If you go to [NV Access](https://www.nvaccess.org) and click Download , you'll land on this download page and be offered to download NVDA 2026.1.

Though, according to [this post](https://www.nvaccess.org/post/nvda-2026-1-1/), the last version is 2026.1.1. And you can download it from this post.

### Why are the other templates not appropriate in this case?
Website issue
### Have you asked for advice on how to report this issue via a community discussion? If so, please link to the discussion
No

### Expected outcome or behaviour (if applicable)
NVDA 2026.1.1 should be offered on NV Access official download page.
---
Cc @Qchristensen

**Comment by seanbudd** [comment]
This is a caching issue. We need to add a step to the release process so we remember to reset our wordpress cache. A hard refresh should fix this on your end

**Comment by seanbudd** [comment]
Added a step in our docs and purged the cache
